[opener] functions are not callable with None as second argument

Pretty straightfoward:

```
app.opener().open_path("whatever", None)
app.opener().open_path("whatever", None)
```

Rust diagnostic:

```
error[E0283]: type annotations needed
   --> src/main.rs:103:44
    |
103 |             app.opener().open_path("path", None);
    |                          ---------         ^^^^ cannot infer type of the type parameter `T` declared on the enum `Option`
    |                          |
    |                          required by a bound introduced by this call
    |
    = note: cannot satisfy `_: Into<std::string::String>`
note: required by a bound in `Opener::<R>::open_path`
   --> /Users/martpie/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tauri-plugin-opener-2.2.1/src/lib.rs:70:27
    |
67  |     pub fn open_path(
    |            --------- required by a bound in this associated function
...
70  |         with: Option<impl Into<String>>,
    |                           ^^^^^^^^^^^^ required by this bound in `Opener::<R>::open_path`
help: consider specifying the generic argument
    |
103 |             app.opener().open_path("path", None::<T>);
    |                                                +++++
```

I had a quick look at the tests, and noticed `None` was not tested (but `safari` or `chrome` was). 

Replacing by `Some("")` opens nothing.

--- a/plugins/opener/README.md
+++ b/plugins/opener/README.md
@@ -98,8 +98,18 @@ fn main() {
         .plugin(tauri_plugin_opener::init())
         .setup(|app| {
             let opener = app.opener();
+
+            // Opens the URL in the default browser
+            opener.open_url("https://example.com", None::<&str>)?;
+            // Or with a specific browser/app
             opener.open_url("https://example.com", Some("firefox"))?;
+
+            // Opens the path with the system's default app
+            opener.open_path("/path/to/file", None::<&str>)?;
+            // Or with a specific app
             opener.open_path("/path/to/file", Some("firefox"))?;
+
+            // Reveal a path with the system's default explorer
             opener.reveal_item_in_dir("/path/to/file")?;
             Ok(())
         })

--- a/plugins/opener/src/lib.rs
+++ b/plugins/opener/src/lib.rs
@@ -38,6 +38,19 @@ pub struct Opener<R: Runtime> {
 impl<R: Runtime> Opener<R> {
     /// Open a url with a default or specific program.
     ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// use tauri_plugin_opener::OpenerExt;
+    ///
+    /// tauri::Builder::default()
+    ///   .setup(|app| {
+    ///     // open the given URL on the system default browser
+    ///     app.opener().open_url("https://github.com/tauri-apps/tauri", None::<&str>)?;
+    ///     Ok(())
+    ///   });
+    /// ```
+    ///
     /// ## Platform-specific:
     ///
     /// - **Android / iOS**: Always opens using default program.
@@ -47,6 +60,19 @@ impl<R: Runtime> Opener<R> {
     }
 
     /// Open a url with a default or specific program.
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// use tauri_plugin_opener::OpenerExt;
+    ///
+    /// tauri::Builder::default()
+    ///   .setup(|app| {
+    ///     // open the given URL on the system default browser
+    ///     app.opener().open_url("https://github.com/tauri-apps/tauri", None::<&str>)?;
+    ///     Ok(())
+    ///   });
+    /// ```
     ///
     /// ## Platform-specific:
     ///
@@ -59,6 +85,19 @@ impl<R: Runtime> Opener<R> {
     }
 
     /// Open a path with a default or specific program.
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// use tauri_plugin_opener::OpenerExt;
+    ///
+    /// tauri::Builder::default()
+    ///   .setup(|app| {
+    ///     // open the given path on the system default explorer
+    ///     app.opener().open_path("/path/to/file", None::<&str>)?;
+    ///     Ok(())
+    ///   });
+    /// ```
     ///
     /// ## Platform-specific:
     ///
@@ -73,6 +112,19 @@ impl<R: Runtime> Opener<R> {
     }
 
     /// Open a path with a default or specific program.
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// use tauri_plugin_opener::OpenerExt;
+    ///
+    /// tauri::Builder::default()
+    ///   .setup(|app| {
+    ///     // open the given path on the system default explorer
+    ///     app.opener().open_path("/path/to/file", None::<&str>)?;
+    ///     Ok(())
+    ///   });
+    /// ```
     ///
     /// ## Platform-specific:
     ///
@@ -98,7 +150,7 @@ pub trait OpenerExt<R: Runtime> {
     fn opener(&self) -> &Opener<R>;
 }
 
-impl<R: Runtime, T: Manager<R>> crate::OpenerExt<R> for T {
+impl<R: Runtime, T: Manager<R>> OpenerExt<R> for T {
     fn opener(&self) -> &Opener<R> {
         self.state::<Opener<R>>().inner()
     }

--- a/plugins/opener/src/open.rs
+++ b/plugins/opener/src/open.rs
@@ -26,7 +26,7 @@ pub(crate) fn open<P: AsRef<OsStr>, S: AsRef<str>>(path: P, with: Option<S>) -> 
 /// tauri::Builder::default()
 ///   .setup(|app| {
 ///     // open the given URL on the system default browser
-///     tauri_plugin_opener::open_url("https://github.com/tauri-apps/tauri", None)?;
+///     tauri_plugin_opener::open_url("https://github.com/tauri-apps/tauri", None::<&str>)?;
 ///     Ok(())
 ///   });
 /// ```
@@ -46,8 +46,8 @@ pub fn open_url<P: AsRef<str>, S: AsRef<str>>(url: P, with: Option<S>) -> crate:
 /// ```rust,no_run
 /// tauri::Builder::default()
 ///   .setup(|app| {
-///     // open the given URL on the system default browser
-///     tauri_plugin_opener::open_path("/path/to/file", None)?;
+///     // open the given URL on the system default explorer
+///     tauri_plugin_opener::open_path("/path/to/file", None::<&str>)?;
 ///     Ok(())
 ///   });
 /// ```

